DashAI/back/api/api_v1/endpoints/runs.py

```diff
@@ -0,0 +1,207 @@
+import logging
+from typing import Union
+
+from fastapi import APIRouter, Depends, Response, status
+from fastapi.exceptions import HTTPException
+from sqlalchemy import exc, select
+from sqlalchemy.orm import Session
+
+from DashAI.back.api.deps import get_db
+from DashAI.back.database.models import Experiment, Run
+
+logging.basicConfig(level=logging.DEBUG)
+log = logging.getLogger(__name__)
+
+router = APIRouter()
+
+
+@router.get("/")
+async def get_run(run_id: int, db: Session = Depends(get_db)):
+    """
+    Returns the run with the spécified id.
+
+    Returns
+    -------
+    JSON
+        run JSON
+    """
+
+    try:
+        run = db.get(Run, run_id)
+        if not run:
+            raise HTTPException(
+                status_code=status.HTTP_404_NOT_FOUND,
+                detail="Run not found",
+            )
+    except exc.SQLAlchemyError as e:
+        log.exception(e)
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail="Internal database error",
+        )
```

Comment:
It is recommended that when you throw an exception in an exception handler (`except`) you use `from` to indicate that the exception you want to throw is caused by the exception caught (`e`) and not by the other code inside the except (such as `log.exception` or other).

```suggestion
        ) from e
```

Reply:
Reference: https://stackoverflow.com/a/24752607